chemprop/nn/predictors.py

```diff
@@ -80,6 +80,7 @@ def __init__(
             input_dim, n_tasks * self.n_targets, hidden_dim, n_layers, dropout, activation
         )
         self.criterion = criterion or self._default_criterion
+        self.freeze = 0
```

Comment:
This is a nitpick, but could we change this to something to reflect that this is a number as `self.freeze` is bool in other locations? Something like `self.freeze_layers`?